process/TAC-Decision-Process.md

```diff
@@ -0,0 +1,25 @@
+# TAC Decision Process #
+
+The OpenSSF Technical Advisory Council (TAC) is a body comprised of 9 individuals that are either elected for a 2 year term or appointed by the governing board for a 1 year term.  
```

Comment:
I thought it was split where some elected folks were also 1 year terms. I don't remember if we changed this to all elected members get 2 year terms.

Reply:
Everyone elected will have a 2year term going forward.  For this first year, where we started the staggering or terms, we asked for volunteers to have a 1year term (and the folks that had served the prior year agreed for this.  Ideally we will capture this in Arnaud's forthcoming awesome TAC election process docs.